lms/static/scripts/frontend_apps/utils/content-item.js

```diff
@@ -5,9 +5,6 @@
 import { stringify } from 'querystring';
 
 /**
- * Return a JSON-LD `ContentItem` representation of the LTI activity launch
- * URL for a given document URL.
- *
```

Comment:
I removed this comment because it is for a different function, `contentItemForUrl`, which already has the comment. The `contentItemWithParams` function is just an internal helper in this module.